src/m365/teams/commands/channel/channel-remove.ts

```diff
@@ -140,14 +130,52 @@ class TeamsChannelRemoveCommand extends GraphCommand {
         type: 'confirm',
         name: 'continue',
         default: false,
-        message: `Are you sure you want to remove the channel ${channelName} from team ${args.options.teamId}?`
+        message: `Are you sure you want to remove the channel ${channelName}?`
```

Comment:
Let's rename the variable `channelName` to `channel` as it can be used for name as well as id.